studio/.prettierrc

```diff
@@ -1,3 +1,5 @@
 {
-  "printWidth": 100
+  "printWidth": 100,
```

Comment:
@elmaanum this is why it's good to use a trailing comma in the last row in objects ;)